Gemfile.lock

```diff
@@ -824,4 +813,4 @@ RUBY VERSION
    ruby 3.0.3p157
 
 BUNDLED WITH
-   2.1.4
+   2.1.4
```

Comment:
Did you edit this manually?

Reply:
I've:
- removed the gems from the `Gemfile`
- ran bundle install
- noticed that `Gemfile.lock` was not being updated - I think it should, right?
- manually deleted the lines in which selenium-webdriver and webdriver + dependencies were mentioned

Reply:
> noticed that Gemfile.lock was not being updated - I think it should, right?

It definitely should. That's weird. I just did the same locally and my Gemfile.lock did get updated.

Did you set the deployment flag for bundler? Maybe look into bundle config and see if you can spot anything odd there.

Reply:
I just ran `bundle` on master and it deleted more indirect dependencies. But it also didn't recognise the bundler version and updated it in the lockfile. So the newline is important for bundler! I'll open a PR for this.

Reply:
https://github.com/openfoodfoundation/openfoodnetwork/pull/9570